package.json

```diff
@@ -7,8 +7,8 @@
   "homepage": "http://www.fluidproject.org/",
   "dependencies": {
     "d3": "3.5.17",
-    "flocking": "colinbdclark/Flocking#6749bee2b7041b76f494bd188b755d568332eda0",
-    "infusion": "2.0.0-dev.20160902T155354Z.b963420"
+    "flocking": "0.2.0-dev.20170206T221943Z.fb09a52",
+    "infusion": "2.0.0"
```

Comment:
It would be great if we can upgrade this all the way to today's dev release - 3.0.0-dev.20170214T161720Z.6126724 - since I am consuming chartAuthoring from "fluid-authoring" where I am relying on these fixes in order to render the "visible nexus" interface